Occasionally failing parser test

```Console
____________________________ test_main_check_quiet _____________________________
[gw1] linux -- Python 3.7.9 /opt/hostedtoolcache/Python/3.7.9/x64/bin/python3

    def test_main_check_quiet() -> None:
>       assert cli.main(["rflx", "-q", "check", "specs/tlv.rflx"]) == 0

tests/test_cli.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
rflx/cli.py:83: in main
    args.func(args)
rflx/cli.py:91: in check
    parse(args.files)
rflx/cli.py:118: in parse
    parser = Parser(skip_verification, cached=True)
rflx/parser/parser.py:63: in __init__
    self.__cache = Cache(cached)
rflx/parser/cache.py:22: in __init__
    self._initialize_cache()
rflx/parser/cache.py:48: in _initialize_cache
    CACHE_DIR.mkdir(parents=True)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = PosixPath('/home/runner/.cache/RecordFlux'), mode = 511, parents = True
exist_ok = False

    def mkdir(self, mode=0o777, parents=False, exist_ok=False):
        """
        Create a new directory at this given path.
        """
        if self._closed:
            self._raise_closed()
        try:
>           self._accessor.mkdir(self, mode)
E           FileExistsError: [Errno 17] File exists: '/home/runner/.cache/RecordFlux'

/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/pathlib.py:1273: FileExistsError
```

```Python
 46     def _initialize_cache() -> None:
 47         if not CACHE_DIR.exists():
 48             CACHE_DIR.mkdir(parents=True)
```

This looks like a race condition. We could fix that by adding `exist_ok=True` to `mkdir`.

--- a/rflx/parser/cache.py
+++ b/rflx/parser/cache.py
@@ -44,8 +44,7 @@ class Cache:
 
     @staticmethod
     def _initialize_cache() -> None:
-        if not CACHE_DIR.exists():
-            CACHE_DIR.mkdir(parents=True)
+        CACHE_DIR.mkdir(parents=True, exist_ok=True)
         if not (CACHE_DIR / VERIFICATION_FILE).exists():
             with open(CACHE_DIR / VERIFICATION_FILE, "w") as f:
                 json.dump({}, f)

--- a/tests/test_cli.py
+++ b/tests/test_cli.py
@@ -9,6 +9,10 @@ from rflx import cli
 from rflx.error import Location, Severity, Subsystem, fail
 
 
+def raise_parser_error() -> None:
+    fail("TEST", Subsystem.PARSER, Severity.ERROR, Location((8, 22)))
+
+
 def raise_model_error() -> None:
     fail("TEST", Subsystem.MODEL, Severity.ERROR, Location((8, 22)))
 
@@ -32,8 +36,9 @@ def test_main_check_quiet() -> None:
     assert cli.main(["rflx", "--quiet", "check", "specs/tlv.rflx"]) == 0
 
 
-def test_main_check_parser_error() -> None:
-    assert "README.md:1:1: parser: error: " in str(cli.main(["rflx", "check", "README.md"]))
+def test_main_check_parser_error(monkeypatch: Any) -> None:
+    monkeypatch.setattr(cli, "check", lambda x: raise_parser_error())
+    assert "<stdin>:8:22: parser: error: TEST" in str(cli.main(["rflx", "check", "README.md"]))
 
 
 def test_main_check_model_error_parse(monkeypatch: Any) -> None:

